src/pages/home/home.module.scss

```diff
@@ -5,3 +5,37 @@
 .title {
   font-weight: bold;
 }
+
+.userCard {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: solid 1px rgba(0, 0, 0, 0.08);
+  cursor: pointer;
+}
+
+.userCard:hover {
```

Comment:
remember to nest the hover behavior inside of the class.
.userCard{
       &:hover {
       }
}